src/riak_kv_qry_worker.erl

```diff
@@ -292,10 +292,8 @@ prepare_final_results2(#riak_sel_clause_v1{col_return_types = ColTypes,
 
 prepare_final_results_test() ->
     Rows = [[12, <<"windy">>], [13, <<"windy">>]],
```

Comment:
`list_to_tuple` now happens in `riak_kv_ts_svc`, right before the records are delivered to the PB/TTB code and further to the client. This is so in order to avoid converting tuples back to lists in `riak_kv_wm_timeseries_query`, where lists are required for converting the data to json.